### DISCUSSION TITLE
Adding link to a portrait

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#224] [Q&A] [answered]
Dear Vincent Rubinetti,

Thank you always for your prompt reply and just the right answers to my questions.

I am planning to add portraits with links to the webpage. For example,

{% include portrait.html name="The James M. Cox Foundation" lookup="https://www.coxenterprises.com/our-impact/james-m-cox-foundation" image="images/sponsors/cox.jpeg" %}

however, the name does not have the link. Is there a trick to this?

**Comment by hyeokhyen** [comment]
I have another question. When using portrait as blow:

{% include portrait.html name="N. Jabin gong (BS, Gatech CS, 2023)" lookup="jabin-gong" image="images/members/jabin_pic.png" style="small" %}

The name is overwritten by jabin-gong. Is there a way to prevent this?

**Reply by vincerubinetti** [reply, marked as answer]
Reading the documentation for this is the best way to understand what to do:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/portrait
https://greene-lab.gitbook.io/lab-website-template-docs/basics/team-members

It seems like you are trying to use the portrait component for a much different purpose than it's designed for. In the first example, you're using a link for `lookup`. Lookup is meant to find a matching **team member** `.md` file and fill in the details (like name, image, etc) from there. In the second example, you're mixing `lookup` and some details like `name` and `image`, when you should only be doing either or.

If you just want to have an circular image that links somewhere (possibly with a caption), you can use the [figure component](https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/figure) and modify its [border radius](https://github.com/greenelab/lab-website-template/blob/main/_styles/figure.scss#L18) to be `9999px`.
